plugins/CoreHome/Columns/VisitorFingerprint.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link http://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreHome\Columns;
+
+use Piwik\Columns\DimensionMetricFactory;
+use Piwik\Columns\MetricsList;
+use Piwik\Piwik;
+use Piwik\Plugin\ArchivedMetric;
+use Piwik\Plugin\Dimension\VisitDimension;
+
+/**
+ * Dimension for the log_visit.config_id column. This column is added in the CREATE TABLE
+ * statement, so this dimension exists only to configure a segment.
+ */
+class VisitorFingerprint extends VisitDimension
+{
+    protected $columnName = 'config_id';
+    protected $metricId = 'visitors';
+    protected $nameSingular = 'General_VisitorFingerprint';
+    protected $namePlural = 'General_Visitors';
+    protected $segmentName = 'fingerprint';
+    protected $acceptValues = '1eceaa833348b187 - any 16 Hexadecimal chars ID, which can be fetched from API.getLastVisitsDetails';
+    protected $allowAnonymous = false;
+    protected $sqlFilterValue = array('Piwik\Common', 'convertVisitorIdToBin');
+
+    public function configureMetrics(MetricsList $metricsList, DimensionMetricFactory $dimensionMetricFactory)
+    {
+        $metric = $dimensionMetricFactory->createMetric(ArchivedMetric::AGGREGATION_UNIQUE);
+        $metric->setTranslatedName(Piwik::translate('Visitor_Fingerprint'));
+        $metric->setName('nb_uniq_visitors_fingerprint');
```

Comment:
Would we still need this call if we set `$metricId` to 'visitors_fingerprint'?

Reply:
When you set `visitors_fingerprint ` as segment name, then the autosuggest doesn't work automatically anymore.

Reply:
I meant as `$metricId`, not `$segmentName`, unless they are related? Not sure I've ever used this API before... :)

Reply:
setting `$metricId` to `visitors_fingerprint` might be a good idea @katebutler  👍 . Shouldn't really change much though. In general the code in that method likely will still be needed to ensure only the `unique fingerprint` metrics is being created.

Reply:
Hi @katebutler, could you please respond to this thread (either by applying the requested change or commenting on why you'd rather not)? Once the thread is resolved I can merge this.